recipes/thrust/all/conanfile.py

```diff
@@ -11,29 +14,29 @@ class ThrustConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     no_copy_source = True
     options = {"device_system": ["cuda", "cpp", "omp", "tbb"]}
-    default_options = {"device_system": "cuda"}
+    default_options = {"device_system": "tbb"}
 
     @property
     def _source_subfolder(self):
         return "source_subfolder"
 
     def requirements(self):
         if self.options.device_system == "tbb":
-            self.requires("tbb/2020.1")
+            self.requires("onetbb/2020.3")
         elif self.options.device_system != "cpp":
             self.output.warn('Conan package for {0} is not available,'
                              ' this package will use {0} from system.'
                              .format(str(self.options.device_system).upper()))
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = self.name + "-" + self.version
-        os.rename(extracted_dir, self._source_subfolder)
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
 
     def package(self):
         self.copy(pattern="LICENSE", src=self._source_subfolder, dst="licenses")
-        self.copy("*", src=os.path.join(self._source_subfolder, "thrust"),
-                  dst=os.path.join("include", "thrust"))
+        self.copy("*.h", src=os.path.join(self._source_subfolder, "thrust"),
+                      dst=os.path.join("include", "thrust"))
+        self.copy("*.inl", src=os.path.join(self._source_subfolder, "thrust"),
+                      dst=os.path.join("include", "thrust"))
```

Comment:
```suggestion
        self.copy("*[.h|.inl]", src=os.path.join(self._source_subfolder, "thrust"),
                      dst=os.path.join("include", "thrust"))
```

Reply:
does it understand that syntax? 👀

Reply:
I was looking for it. Conan uses fnmatch to calculate that pattern, but I didn't find a relation with multiple patterns. Thank you!